source/_data/projects.yml

```
@@ -0,0 +1,33 @@
+# open source projects that use Cypress to write end-to-end tests
+
+- name: Redash
+  description: "Connect and query your data sources, build dashboards to visualize data and share them with your company."
+  url: https://github.com/getredash/redash/tree/master/client
+
+- name: Mermaid.js
+  description: "Generation of diagram and flowchart from text in a similar manner as markdown"
+  url: https://github.com/mermaid-js/mermaid
+
+- name: WithSpectrum
```

This repo is called 'Spectrum' not 'WithSpectrum'
```suggestion
- name: Spectrum
```